app/policies/resource_policy.rb

```diff
@@ -0,0 +1,12 @@
+class ResourcePolicy < ApplicationPolicy
```

Comment:
I presume this class will have edit and other types of permissions once we develop it further, yes?